ndjson-join

```diff
@@ -25,6 +27,8 @@ if (commander.args.length < 4) {
   commander.args.splice(1, 0, commander.args[0]);
 }
 
+var join = commander.left ? leftJoin : commander.right ? rightJoin : innerJoin;
+
```

Comment:
So if both `--left` and `--right` are set, `--left` takes priority? Perhaps instead, if both are set we should do an outerJoin?